ffi/capi/src/provider.rs

```diff
@@ -176,39 +176,8 @@ pub mod ffi {
             Ok(())
         }
 
-        /// Enables locale fallbacking for data requests made to this provider.
-        ///
-        /// Note that the test provider (from `create_test`) already has fallbacking enabled.
-        #[diplomat::rust_link(
-            icu_provider_adapters::fallback::LocaleFallbackProvider::try_new,
-            FnInStruct
-        )]
-        #[diplomat::rust_link(
-            icu_provider_adapters::fallback::LocaleFallbackProvider,
-            Struct,
-            compact
-        )]
-        pub fn enable_locale_fallback(&mut self) -> Result<(), ICU4XDataError> {
```

Comment:
Not 100% sure we want to bring the change to FFI since the alternative may be less efficient (an extra allocation and FFI call), but it seems like something we could add back later if needed

Reply:
Fewer mutable methods over FFI is better anyway